test/common-utils/configure_zsh_as_default_shell.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+# Optional: Import test library
+source dev-container-features-test-lib
+
+# Definition specific tests
+check "default-shell-is-zsh" getent passwd vscode | awk -F: '{ print $7 }' | grep "/bin/zsh"
```

Comment:
I've hardcoded `vscode` here as its the default user created by common-utils. Ideally I'd not want to hardcode this, but the logic for getting the user isn't the nicest

Reply:
Left a comment on how you could possibly avoid hardcoding `vscode`